crates/policy-testdrive/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Is this LICENSE mandatory here or can be assumed inheriting the root one?

Reply:
Probably not, I'll drop it